lightwood/mixers/nn/nn.py

```diff
@@ -320,7 +325,9 @@ def predict(self, when_data_source, include_extra_data=False):
 
             with torch.no_grad():
                 if self.is_selfaware:
-                    output, awareness = self.net(inputs)
+                    self.selfaware_net.eval()
```

Comment:
it seems like the main network is not set to `eval`, if so we should also do this here. If it is set, but somewhere else, we should set the selfaware to eval there as well.

Reply:
Same as in the previous comment